theme/css/swiper.css

```diff
@@ -0,0 +1,50 @@
+/*testimonial */
+.testimonial-container.swiper-container {
+  width: 100%;
+  height: 450px;
+  padding-top: 10px;
+}
+
+.testimonial-container.swiper-container.full-width {
```

Comment:
This part of the styling should be in the style.css file imo.

Reply:
okay